x-pack/plugins/security_solution/common/endpoint/models/event.ts

```diff
@@ -9,10 +9,15 @@ export function isLegacyEvent(event: ResolverEvent): event is LegacyEndpointEven
   return (event as LegacyEndpointEvent).endgame !== undefined;
 }
 
-export function isProcessStart(event: ResolverEvent): boolean {
+export function isStart(event: ResolverEvent): boolean {
   if (isLegacyEvent(event)) {
     return event.event?.type === 'process_start' || event.event?.action === 'fork_event';
   }
+
+  if (Array.isArray(event.event.type)) {
+    return event.event.type.length === 1 && event.event.type[0] === 'start';
```

Comment:
:red_circle: https://www.elastic.co/guide/en/ecs/current/ecs-event.html The `type` entry doesn't say anything about `start` having to come first or the array needing to have a length of one. Should we check instead to see that it .includes `start`

Reply:
Yeah good point 👍  I'll update it.